pkg/pmem-device-manager/pmd-lvm.go

```diff
@@ -158,12 +158,18 @@ func (lvm *pmemLvm) DeleteDevice(name string, flush bool) error {
 	if err != nil {
 		return err
 	}
-	if err := ClearDevice(device, flush); err != nil {
+	if err = ClearDevice(device, flush); err != nil {
 		return err
 	}
 
-	_, err = pmemexec.RunCommand("lvremove", "-fy", device.Path)
-	return err
+	if _, err = pmemexec.RunCommand("lvremove", "-fy", device.Path); err != nil {
```

Comment:
Same here.

Forget about "but reusing the same variable is more efficient" - it's impossible to tell whether that is measurable or even true. The compiler might do a lifecycle analysis and reuse the exact same memory automatically.